examples/all_plugins/android/app/build.gradle

```diff
@@ -35,10 +37,11 @@ android {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "io.plugins.all_plugins"
         minSdkVersion 16
+        multiDexEnabled true
```

Comment:
The Gradle changes here and below look good, but they're strictly unrelated to the stack tracing so I'd prefer to split them out in case there's a need for rolling back for any odd reason. WDYT?

Reply:
These were unrelated changes not authored by me, and when I merged the latest master they went away. I just intended to add the stack trace thing.